### PR TITLE
PP-15 Add export table to Excel spreadsheet button

### PR DESCRIPTION
**Pull request by ntsim**
This PR introduces [SheetJS](https://github.com/SheetJS/sheetjs) to export our rendered data tables into Excel spreadsheet format. Interestingly, SheetJS is able to convert raw HTML tables directly into spreadsheets!

## Other changes
- Replaces the current export to CSV button's implementation (using Papaparse) with SheetJS as it can also handle creation of CSVs.

### CODE REVIEW

File: src/explore-education-statistics-common/src/modules/table-tool/components/DownloadExcelButton.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    sheet[`A${footnoteStartRow + index}`] = {
+      t: 's',
+      v: `(${index + 1}) ${footnote.label}`,
+    } as CellObject;
+  });
+
+  // eslint-disable-next-line no-param-reassign
+  sheet['!ref'] = `A1:${lastColumn}${footnoteStartRow + footnotes.length - 1}`;
+
+  return sheet;
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-common/src/modules/table-tool/components/DownloadExcelButton.tsx]: Sweeeeeeet!

File: src/explore-education-statistics-admin/config/webpack.config.js [hunk 2]
```diff
@@ -270,6 +270,7 @@ module.exports = webpackEnv => {
         .map(ext => `.${ext}`)
         .filter(ext => useTypeScript || !ext.includes('ts')),
       alias: {
+        './dist/cpexcel.js': '',
```
[the remark this answers is not in the document]
Comment (ntsim) [reply src/explore-education-statistics-admin/config/webpack.config.js]: This is just from the installation instructions for SheetJS. We're just removing some optional dependency from the bundle that we don't need:

https://github.com/SheetJS/sheetjs/tree/master/demos/webpack#omitting-optional-dependencies

[the remark this answers is not in the document]
Comment (duncan-at-hiveit) [reply src/explore-education-statistics-admin/config/webpack.config.js]: Cool ta

**Comment by duncan-at-hiveit** [comment]
Hey @ntsim just need to resolve merge conflict, then happy for you to merge in

**Pull request merged**